crates/e2e/tests/e2e/limit_orders.rs

```diff
@@ -446,3 +465,88 @@ async fn too_many_limit_orders_test(web3: Web3) {
     assert_eq!(status, 400);
     assert!(body.contains("TooManyLimitOrders"));
 }
+
+async fn forked_single_limit_order_test(web3: Web3) {
+    let mut onchain = OnchainComponents::deployed(web3.clone()).await;
+    let forked_node_api = web3.api::<ForkedNodeApi<_>>();
+
+    let [solver] = onchain.make_solvers_forked(to_wei(1)).await;
+
+    let [trader] = onchain.make_accounts(to_wei(1)).await;
+
+    let token_usdc = ERC20::at(
+        &web3,
+        "0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48"
+            .parse()
+            .unwrap(),
+    );
+
+    let token_usdt = ERC20::at(
+        &web3,
+        "0xdac17f958d2ee523a2206206994597c13d831ec7"
+            .parse()
+            .unwrap(),
+    );
+
+    // Give trader some USDC
+    let usdc_whale = forked_node_api.impersonate(&USDC_WHALE).await.unwrap();
+    tx!(
+        usdc_whale,
+        token_usdc.transfer(trader.address(), to_wei_with_exp(1000, 6))
+    );
+
+    // Approve GPv2 for trading
+    tx!(
+        trader.account(),
+        token_usdc.approve(onchain.contracts().allowance, to_wei_with_exp(1000, 6))
+    );
```

Comment:
Just a small detail but wouldn't it be simpler to directly create the order using the impersonated whale?
Feel free to ignore, though.

Reply:
We can't sign the order details with the whale though because we don't have their private key. Instead I guess we'd need to do an onchain tx to create the Cow order, but seeing as that's not the default flow for most orders, I think it's best do it this way for now and have another test in future which does the onchain order. What do you think?

Reply:
Ahh, I see. We can transfer tokens from the whale to the trader because it's impersonated and everything happens inside the forked EVM. But to place an order we'd actually have to sign the offchain message which requires the private key of the whale outside the EVM.
Makes sense. 👍 
I agree that the current version tests the more common flow so we can leave it as it.